client/app/telecom/telephony/line/management/mgcpIpRestriction/telecom-telephony-line-mgcpIpRestriction.js

```diff
@@ -0,0 +1,14 @@
+angular.module("managerApp").config(function ($stateProvider) {
+    "use strict";
+    $stateProvider.state("telecom.telephony.line.mgcpIpRestriction", {
+        url: "/mgcpIpRestriction",
+        views: {
+            "@lineView": {
+                templateUrl: "app/telecom/telephony/line/management/mgcpIpRestriction/telecom-telephony-line-mgcpIpRestriction.html",
+                controller: "TelecomTelephonyLineMgcpIpRestrictionCtrl",
+                controllerAs: "MgcpIpRestrictionCtrl"
+            }
+        },
+        translations: ["common", "telecom/telephony/line/management/mgcpIpRestriction"]
```

Comment:
`"common"` translations are already loaded by default, it's not needed to put it here :)

Check here : https://github.com/ovh-ux/ovh-manager-telecom/blob/master/client/app/telecom/telecom.js#L9

Telecom load it, and this state is a child of telecom, so smaller better faster stronger to remove it here (8)